### PR TITLE
chore(release): version 2.0.0

### PR DESCRIPTION
**Pull request by pallabmaiti**
## Description

> Pulled the latest from Segment and added required changes.

### CODE REVIEW

**Review by desusai7 — APPROVED** [review]

File: lib/rudder/analytics/defaults.rb [hunk 1]
```diff
@@ -7,11 +7,9 @@ module Request
         HOST = 'localhost'
         PORT = 8080
         PATH = '/v1/batch'
-        DATA_PLANE_URL = 'http://localhost:8080/v1/batch'
-        SSL = false
         HEADERS = { 'Accept' => 'application/json',
                     'Content-Type' => 'application/json',
-                    'User-Agent' => "rudderanalytics-ruby/#{Analytics::VERSION}" }
```
Comment (bardisg) [inline lib/rudder/analytics/defaults.rb:14]: is this info now part of the event.context.library?

File: lib/rudder/analytics/field_parser.rb [hunk 2]
```diff
@@ -135,40 +129,64 @@ def parse_for_screen(fields)
...
 
-          parsed[:userId] = fields[:user_id] if fields[:user_id]
-          parsed[:anonymousId] = fields[:anonymous_id] if fields[:anonymous_id]
-          parsed[:integrations] = fields[:integrations] if fields[:integrations]
-
-          # Not in spec, retained for backward compatibility
-          parsed[:options] = fields[:options] if fields[:options]
-
+          # add the userId if present
+          parsed = parsed.merge({ :userId => fields[:user_id] }) if fields[:user_id]
```
Comment (bardisg) [inline lib/rudder/analytics/field_parser.rb:153]: ensure userId value is string

File: lib/rudder/analytics/field_parser.rb [hunk 3]
```diff
@@ -135,40 +129,64 @@ def parse_for_screen(fields)
...
+
+          # add the anonymousId if present
+          parsed = parsed.merge({ :anonymousId => fields[:anonymous_id] }) if fields[:anonymous_id]
```
Comment (bardisg) [inline lib/rudder/analytics/field_parser.rb:156]: ensure anonimousId value is string

**Comment by bardisg** [comment]
can we also add sonar?

File: bin/analytics_rudder.rb [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+require 'rudder/analytics'
+require 'rubygems'
+require 'commander/import'
+require 'time'
+require 'json'
+require 'yaml'
+
+program :name, 'simulator.rb'
+program :version, '0.0.1'
```
[the remark this answers is not in the document]
Comment (pallabmaiti) [reply bin/analytics_rudder.rb:13]: This file has nothing to do with the SDK. We can ignore it.

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Description
```
[the remark this answers is not in the document]
Comment (pallabmaiti) [reply .github/pull_request_template.md:1]: I used whatever we were using for Go, and Ruby(Sync).

File: .github/workflows/actions.yml [hunk 6]
```diff
@@ -1,33 +0,0 @@
...
-        uses: actions/setup-ruby@v1
-        with:
-          ruby-version: 2.6.x
-
-      - name: Run specs
-        run: |
-          bundle install
-          rake spec
-
-      - name: Build and Publish
```
[the remark this answers is not in the document]
Comment (pallabmaiti) [reply .github/workflows/actions.yml:23]: This id Segment's CI. I deleted it.

File: lib/rudder/analytics/defaults.rb (hunk 1 above)
Comment (pallabmaiti) [reply lib/rudder/analytics/defaults.rb:14]: We don't send User-Agent manually in any other SDK. This line is removed.

**Review by bardisg — APPROVED** [review]

**Pull request merged**